galaxy_ng/app/access_control/statements/roles.py

```diff
@@ -1,67 +1,112 @@
-VIEWSETS = {
-    "CollectionViewSet": {
-        "LOCKED_ROLES": {
-            "galaxy.collection_admin": [
-                "galaxy.change_namespace",
-                "galaxy.delete_namespace",
-                "galaxy.view_namespace",
-                "galaxy.upload_to_namespace",
-                "ansible.delete_collection",
-            ],
-        }
+LOCKED_ROLES = {
+    "galaxy.collection_admin": {
+        "permissions": [
+            "galaxy.add_namespace",
+            "galaxy.change_namespace",
+            "galaxy.delete_namespace",
+            "galaxy.upload_to_namespace",
+            "ansible.delete_collection",
+            "ansible.change_collectionremote",
+            "ansible.view_collectionremote",
+            "ansible.modify_ansible_repo_content",
```

Comment:
This PR also adds `ansible.modify_ansible_repo_content` to `galaxy.collection_admin`. I assume we still want the role `galaxy.content_admin` (but it wouldnt be needed for CRC). We can remove `galaxy.content_admin` from `maintain-pe-group.py` or keep it there as it just overlaps a perm with `galaxy.collection_admin`

I also see we could remove `galaxy.namespace_owner` from `maintain-pe-group.py` since it overlaps perms with `galaxy.collection_admin`.

Let me know if you want me to do a PR to remove these on this branch.

Reply:
`galaxy.content_admin` seems like it should include EEs and collections maybe. @awcrosby what do you think?

Reply:
I'm trying to think where to draw the line between `galaxy.collection_admin` and `galaxy.content_admin`. Would it be more clear to move the 4 namespace perms (add/change/delete/upload) to a namespace_admin role?

Then if kinda feels like we have 2 roles left (maybe could be renamed?):
* Role `galaxy.collection_delete`
  * `ansible.delete_collection`
* Role `galaxy.collection_edit`
  * `ansible.modify_ansible_repo_content` - Move a collection already in the system across repos
  * `ansible.view|change_collectionremote` - Bring in collections from remotes

Do we want to mix collections and non-collection perms in the locked roles?, im not sure we know enough yet about how they might be used to create a good mix right now.